packages/application-to-register-webapp/src/views/land/add-legal-agreement-parties.html

```diff
@@ -83,7 +83,6 @@ <h1 class="govuk-heading-l" tabindex="-1">
                                     name: "organisation[" + i + "][role]",
                                     value: "County Council",
                                     text: "County Council",
-                                    label: "County Council",
                                     checked: roleRowIndex.county_council,
                                     classes: "govuk-input--width-2",
                                     attributes: {
```

Comment:
The inputs data-name and data-id attributes aren't set correctly, think they need the same concatenation as the id/name fields. As it comes through to the HTML as "%index%" . I realise they're not part of the bug, but probably should be fixed